tests/typed_ir/T574.links

```diff
@@ -0,0 +1,4 @@
+#Unchanged from prelude
```

Comment:
You need to register this test in ` tests/typed_ir.tests`. At the moment it's not being run by the testsuite.

Reply:
Good catch, thanks!